php-classes/Slate/Connectors/DataWarehouse/Connector.php

```diff
@@ -394,7 +394,7 @@ protected static function retry(IJob $Job,  callable $callable, $expectedErrors,
             }
 
             $Job->log(
-                LogLevel::ERROR,
+                LogLevel::WARNING,
                 'Error: {errorMessage}',
                 [
                     'errorMessage' => $e->getMessage()
```

Comment:
Maybe we should log the event as an error just before line # 416 when the max retries have been used.